driver-cqlverify/src/main/java/io/nosqlbench/activitytype/cqlverify/CqlVerifyActivity.java

```diff
@@ -22,27 +23,26 @@ public CqlVerifyActivity(ActivityDef activityDef) {
     @Override
     public synchronized void initActivity() {
 
-        if (!super.getActivityDef().getParams().contains("verify") &&
-            !super.getActivityDef().getParams().contains("verify-fields")) {
```

Comment:
`ConcurrentHashMap.contains` calls `containsValue` internally... so this never used to work.
what is the impact of now properly setting these param defaults?

Reply:
The logic would force all verify activities to use params verify=* and compare=all, which is the more conservative setting. It would have been a nuisance, because it wouldn't let a user relax the verify constraints.

Reply:
not sure i understand... my suggested changes here already change these to use `containsKey` instead

Reply:
Yep, I glossed over that detail!